.github/actions/setup/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Setup
+description: Setup Node.js and install dependencies
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version-file: .nvmrc
+    
+    - name: Setup Java
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'zulu'
+        java-version: '17'
+
+    - name: Setup Swift
+      uses: swift-actions/setup-swift@v2
+      with:
+        swift-version: '5.10'
+
+    - name: Setup Swiftlint
+      run: |
+        curl -sL https://github.com/realm/SwiftLint/releases/latest/download/swiftlint_linux.zip -o swiftlint.zip
```

Comment:
I believe it is not a good practice to download (at least the latest version) every time the configuration is performed.
It could be that some parameter changes and starts to break, generating a false positive that we are having some type of problem.

I think we can pin a version or some other type of solution (like adding it to our code or even having a tool repository and pulling it from there)

Reply:
Hi @jose-pereira-caf, I change the URL for specific version. 
URL: `https://github.com/realm/SwiftLint/releases/download/0.54.0/swiftlint_linux.zip`